backend/src/AuthService/AuthService.Api/Program.cs

```diff
@@ -1,6 +1,7 @@
 using AuthService.Application;
 using AuthService.Infrastructure;
 using AuthService.Persistence;
+using Microsoft.EntityFrameworkCore;
 
 var builder = WebApplication.CreateBuilder(args);
 builder.WebHost.UseUrls("http://0.0.0.0:8082");
```

Comment:
Whats this url `http://0.0.0.0:8082` and its purpose? Move it to appsettings and name properly so that it clear what it is